src/Kentico.Xperience.AzureSearch/Admin/UIPages/IndexListingPage.cs

```diff
@@ -177,6 +201,24 @@ public async Task<ICommandResponse<RowActionResult>> Rebuild(int id, Cancellatio
             return ResponseFrom(result)
                 .AddSuccessMessage("Indexing in progress. Visit your AzureSearch dashboard for details about the indexing process.");
         }
+        catch (Azure.RequestFailedException ex)
```

Comment:
Why is the exception handling in the UI and not in DefaultAzureSearchClient?

Reply:
Still, it should be done a better way as described there: https://github.com/Kentico/xperience-by-kentico-azure-ai-search/pull/76/files#r2431195899